hs/src/Reach/Connector/ALGO.hs

```diff
@@ -1016,6 +1016,7 @@ cprim = \case
   BAND -> call "&"
   BIOR -> call "|"
   BXOR -> call "^"
+  DIGEST_XOR -> call "b^"
```

Comment:
You have to adjust the cost analysis for this --- https://developer.algorand.org/docs/get-details/dapps/avm/teal/opcodes/#b_12

Reply:
Is it not already handled by this? https://github.com/reach-sh/reach-lang/blob/946d738daaf83e25002475550c473bf3ed516dbe/hs/src/Reach/Connector/ALGO.hs#L530